src/completions/History.ts

```diff
@@ -58,8 +59,16 @@ export class HistoryCompletionSource extends Completions.CompletionSourceFuse {
             return
         }
 
+        if (prefix == "tabopen ") {
+            if (query.startsWith("-c")) {
+                let args = query.split(" ")
+                options = args.slice(0, 2).join(" ")
+                query = args.slice(2).join(" ")
+            }
+        }
```

Comment:
This should really be done programmatically with the mythical registering API, but I agree that this is good enough until that appears. It would probably be worth leaving a comment that says as much for future archeologists.

Do you fancy doing one for `winopen -private` while you're here?